airflow/providers/google/suite/hooks/drive.py

```diff
@@ -159,6 +160,46 @@ def exists(
             )
         )
 
+    def _resolve_file_path(self, file_id: str) -> str:
+        """
+        Returns the full Google Drive path for given file_id
+
+        :param file_id: The id of a file in Google Drive
+        :return: Google Drive full path for a file
+        """
+        MAX_NESTED_FOLDERS_LEVEL = 20
```

Comment:
Wouod be great to link to the docs as comment here so that it's not seen to be as magical as it looks.